### PR TITLE
(Feat) ‘Virtualisation system’ criterion in the rules

### PR DESCRIPTION
**Pull request by Lainow**
## Checklist before requesting a review

*Please delete options that are not relevant.*

- [x] I have read the CONTRIBUTING document.
- [x] I have performed a self-review of my code.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [ ] This change requires a documentation update.

## Description

Addition of a new ‘Virtualisation system’ criterion in the rules for importing and linking equipment (for Computers). (Ticket : !33353)

## Screenshots (if appropriate):
image

### CODE REVIEW

File: src/Glpi/Inventory/Asset/VirtualMachine.php
Comment (trasher) [inline src/Glpi/Inventory/Asset/VirtualMachine.php:315]: Is this really needed?

File: phpunit/functional/Glpi/Inventory/InventoryTest.php
Comment (trasher) [inline phpunit/functional/Glpi/Inventory/InventoryTest.php:8476]: There is no "db" vm.

Also, as far as I've understood, this rule is to prevent import of "docker" VMs, but all types from example are docker and are imported. 
Did I miss something?

(I've jsut checked this test for now).

File: phpunit/functional/Glpi/Inventory/InventoryTest.php
Comment (trasher) [inline phpunit/functional/Glpi/Inventory/InventoryTest.php:8728]: It des not seems required to have a different agent; just using the same JSON should work the same.

Also, I do not really understand what the update really does from the VMs point of view. It checks nothing has changed; but I guess nothing would have changed without new rules added.

File: phpunit/functional/Glpi/Inventory/InventoryTest.php
Comment (trasher) [inline phpunit/functional/Glpi/Inventory/InventoryTest.php:8731]: Wrong comment. VMs have been created at the first inventory.

File: phpunit/functional/Glpi/Inventory/InventoryTest.php
Comment (trasher) [inline phpunit/functional/Glpi/Inventory/InventoryTest.php]: You must build a new json object, the existing one is currently modified by the inventory process (even if it should not, I did not take time to investigate since it's not a big deal).

But we'll talk about that tomorrow in our internal channel ;)

**Review by stonebuzz — APPROVED** [review]
seems ok to me, let's wait for the customer's feedback

**Review by cedric-anne — CHANGES_REQUESTED** [review]
Tests are failing.

**Comment by trasher** [comment]
Tests are still failing

**Review by trasher — APPROVED** [review]
LGTM, we should wait for customer validation before merge

**Comment by cedric-anne** [comment]
I changed the status to draft as we are still waiting for the customer's feedback.

@Lainow Please warn us when all will be OK for the customer.

**Pull request merged**
